docs/en/deployment.md

```diff
@@ -0,0 +1,38 @@
+Before you launch the solution, review the architecture, supported regions, and other considerations discussed in this guide. Follow the step-by-step instructions in this section to configure and deploy the solution into your account.
```

Comment:
we can list all prerequisites in a separate section, with sub sections include specialized knowledge, assumption, resource requirement (supported region, service quota etc.)